pyatlan/client/aio/client.py

```diff
@@ -134,6 +135,34 @@ def __init__(self, **kwargs):
         # Initialize sync client (handles all validation, env vars, etc.)
         super().__init__(**kwargs)
 
+        if self.oauth_client_id and self.oauth_client_secret and self.api_key is None:
+            LOGGER.debug(
+                "API Key not provided. Using Async OAuth flow for authentication"
+            )
+            from pyatlan.client.aio.oauth import AsyncOAuthTokenManager
```

Comment:
can we move this import at top of the file (hoping we don't get any circular import error 🤞)